examples/deployment/src/asa-helpers.js

```diff
@@ -0,0 +1,163 @@
+  // We will account for changing transaction parameters
+  // before every transaction in this example
+  params = await deployer.algodClient.getTransactionParams().do();
+  //comment out the next two lines to use suggested fee
+  params.fee = 1000;
+  params.flatFee = true;
+
+  let sender = optInAccount.addr;
+  let recipient = sender
+  // We set revocationTarget to undefined as
+  // This is not a clawback operation
+  let revocationTarget = undefined;
+  // CloseReaminerTo is set to undefined as
+  // we are not closing out an asset
+  let closeRemainderTo = undefined;
+  // We are sending 0 assets
+  const note = undefined;
+
+  // transferring 0 will enable future transfers
+  const amount = 0;
+
+  // signing and sending "txn" allows sender to begin accepting asset specified by creator and index
+  let opttxn = algosdk.makeAssetTransferTxnWithSuggestedParams(sender, recipient, closeRemainderTo, revocationTarget,
+    amount, note, assetID, params);
+
+  // Must be signed by the account wishing to opt in to the asset
+  rawSignedTxn = opttxn.signTxn(optInAccount.sk);
+  let opttx = (await deployer.algodClient.sendRawTransaction(rawSignedTxn).do());
+  console.log("ASA Opt-in:", {
+    forAccount: sender,
+    assetID: assetID,
+    txId: opttx.txId
+  })
+  // wait for transaction to be confirmed
+  return await deployer.waitForConfirmation(opttx.txId);
+
+  ////You should now see the new asset listed in the account information
+  //console.log("Master = " + master);
+  //await printAssetHolding(algodclient, master, assetID);
+}
+
+exports.transferAsset = async function (deployer, assetID, fromAccount, toAccountAddr, amount) {
+  // Transfer New Asset:
+
+  // First update changing transaction parameters
+  // We will account for changing transaction parameters
+  // before every transaction in this example
+
+  params = await deployer.algodClient.getTransactionParams().do();
+  //comment out the next two lines to use suggested fee
+  params.fee = 1000;
+  params.flatFee = true;
+
+  sender = fromAccount;
+  recipient = toAccountAddr;
+  revocationTarget = undefined;
+  closeRemainderTo = undefined;
+  note = undefined;
+  //Amount of the asset to transfer
+
+  // signing and sending "txn" will send "amount" assets from "sender" to "recipient"
+  let xtxn = algosdk.makeAssetTransferTxnWithSuggestedParams(sender.addr, recipient, closeRemainderTo, revocationTarget,
+    amount,  note, assetID, params);
+  // Must be signed by the account sending the asset
+  rawSignedTxn = xtxn.signTxn(sender.sk)
+  let xtx = (await deployer.algodClient.sendRawTransaction(rawSignedTxn).do());
+  console.log("Transferring:", {
+    from: sender.addr,
+    to: recipient,
+    amount: amount,
+    assetID: assetID,
+    txId: xtx.txId
+  })
+  // wait for transaction to be confirmed
+  await deployer.waitForConfirmation(xtx.txId);
+
+  //console.log("Master Account = " + master);
+  //await printAssetHolding(deployer.algodClient, master, assetID);
+}
```

Comment:
how about adding most of this functions to `algob` package? Eg - they can be in `src/lib/printers.ts`

The transfer functions also should be in `algob` but in other module.

Reply:
What about later?